.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: d
-services:
-  - docker
```

Comment:
Interesting that Travis is still green without this -> Docker is enabled by default.